functional/pipeline.py

```diff
@@ -187,6 +187,14 @@ def _transform(self, *transforms):
                 sequence = Sequence(self, transform=transform)
         return sequence
 
+    def transform(self, *transforms):
```

Comment:
Is there a reason to make this public? The usage in code calls `self._transform` which I think is fine?

If there is a strong reason to make this public, then the docstring should explicitly reference https://github.com/EntilZha/PyFunctional/blob/master/functional/transformations.py#L15

Reply:
Actually, I made it public because of Pylint warning (even if I made the decorator as a class method). I could disable the warning, it would be cleaner I think.